lib/signet/oauth_2/client.rb

```diff
@@ -823,6 +828,24 @@ def expires_at= new_expires_at
         @expires_at = normalize_timestamp new_expires_at
       end
 
+      ##
+      # Returns the scopes granted by the authorization server.
+      #
+      # @return [Array] The scope of access returned by the authorization server.
```

Comment:
It looks like this can return `nil` as well as `Array`, if the granted scopes were never set.